lifemonitor/auth/templates/auth/apikeys_tab.j2

```diff
@@ -1,8 +1,28 @@
-<div class="m-2 p-2">
-    Allow a user to interact with the LifeMonitor API.
-    An API key acts as a static authentication token
-    that can be used to quickly
-    try API calls via the API docs interface or tools like <span class="text-monospace">curl</span>.
+<div class="row" style="margin-top: 20px;">
+    <div class="col-sm-9 col-lg-10 mt-2 p-4">
+        Allow a user to interact with the LifeMonitor API.
+        An API key acts as a static authentication token
+        that can be used to quickly
+        try API calls via the API docs interface or tools like <span class="text-monospace">curl</span>.
+        asdasad
+    </div>
+    <div class="col-sm-3 col-lg-2 d-block">
+        <div>
+            <a href="/openapi.html" title="LifeMonitor OpenAPI Explorer">        
```

Comment:
I think this should read "LifeMonitor API Explorer". The fact that the API follows the OpenAPI standard is a detail, what matters is that at that link you can find the API docs and try the calls.

Reply:
Fixed in https://github.com/crs4/life_monitor/pull/173/commits/961e3f0accaef52abcef13033686ebb0272da1d7